doc/whats-new.rst

```diff
@@ -13,6 +13,19 @@ What's New
     import xarray as xr
     np.random.seed(123456)
 
+.. _whats-new.0.9.1:
+
+v0.9.1 (unreleased)
+
+Enhancements
+~~~~~~~~~~~~
+
+- Added the xarray equivalent of `pandas.Dataframe.combine_first` as an instance
```

Comment:
Actually let's move this into v0.9.0. I think this change is pretty safe, and it would be nice to get it in promptly.